packages/web/src/components/common/my-position-card-list/MyPositionCardList.tsx

```diff
@@ -38,8 +38,8 @@ const MyPositionCardList: React.FC<MyPositionCardListProps> = ({
   themeKey,
   showLoadMore,
 }) => (
-  <CardListWrapper loading={isLoading}>
-    <GridWrapper ref={divRef} onScroll={onScroll} loading={isLoading}>
+  <CardListWrapper $loading={isLoading}>
```

Comment:
Why did you change it to `$loading`?
Is it to express a property of the styled component?

Reply:
this is a fix for style component warning, it's a transition props
please check for more detail: https://styled-components.com/docs/faqs#why-am-i-getting-html-attribute-warnings